tests/weights/test_dweights_precalibration_checkpoints.py

```diff
@@ -51,23 +43,22 @@ def test_precal_and_design_weights_checkpoints(spark_session):
             country_column="country",
             rounding_value=18,
         )
+        assert "Antibody design weights do not sum to country population totals" in weightexception.value
```

Comment:
perhaps use: assert all([x in weightexception.value for x in [%all off the options%]])